commons/src/main/java/com/droidcon/commons/sessionize/storage/database/SessionizeDatabase.kt

```diff
@@ -0,0 +1,32 @@
+package com.droidcon.commons.sessionize.storage.database
```

Comment:
Don't have a really strong opinion on this. And current naming and package structure its fine. But I'm not a big fan of coupling db naming nor package structure to the naming of a web site that provides the service. I would rather go for more semantic naming that better reflect each thing or that follow known patterns in package structure that make navigating through code more intuitive